app/connectors/NsiConnector.scala

```diff
@@ -38,7 +39,7 @@ class NsiConnector @Inject() (
     httpClient: HttpClientV2,
     servicesConfig: ServicesConfig
   )(implicit ec: ExecutionContext
-  ) extends BackendHeaderCarrierProvider {
+  ) extends BackendHeaderCarrierProvider with FormattedLogging {
```

Comment:
If you want to use static constants for standard headers like `AUTHORIZATION`, mixing in the trait `HeaderNames` on this line.